tests/pipeline/test_buyback_auth.py

```diff
@@ -150,13 +140,16 @@ def setup(self, dates):
         cols = {}
         cols[
```

Comment:
How do you feel about these being of the form `cols[NAME] = \` with the rest on the line below? I know the form is to try to keep from doing a line continuation, but in this case, I actually got confused about what I was looking at because of where the brackets were.